src/scf/mixing.jl

```diff
@@ -85,16 +89,16 @@ from the current density of states at the Fermi level.
     α::Real = 0.8
 end
 @timing "KerkerDosMixing" function mix(mixing::KerkerDosMixing, basis::PlaneWaveBasis,
-                                       δF, δFspin=nothing; εF, ψ, eigenvalues, kwargs...)
+                                       δF; εF, ψ, eigenvalues, kwargs...)
     if basis.model.temperature == 0
-        return mix(SimpleMixing(α=mixing.α), basis, δF, δFspin)
+        return mix(SimpleMixing(α=mixing.α), basis, δF)
     else
         n_spin = basis.model.n_spin_components
         Ω = basis.model.unit_cell_volume
-        dos  = [compute_dos(εF, basis, eigenvalues, spins=[σ]) / Ω for σ in 1:n_spin]
+        dos  = compute_dos(εF, basis, eigenvalues)
         kTF  = sqrt(4π * sum(dos))
-        ΔDOS = n_spin == 2 ? dos[1] - dos[2] : 0.0
-        mix(KerkerMixing(α=mixing.α, kTF=kTF, ΔDOS=ΔDOS), basis, δF, δFspin)
+        ΔDOS = n_spin == 2 ? dos[1]-dos[2] : 0.0
```

Comment:
you really don't like spaces do you?

Reply:
wasn't on purpose, probably I deleted it and added back in again, forgetting about the spaces. However this is typically a case where no spaces makes the structure a lot clearer

Reply:
depends what your eye is used to. I find the original easier to parse.